src/Core/Domain/Category/SeoSettings.php

```diff
@@ -26,28 +26,13 @@
 
 namespace PrestaShop\PrestaShop\Core\Domain\Category;
 
+use PrestaShop\PrestaShop\Core\Domain\Meta\SeoSettings as ParentSeoSettings;
+
 /**
  * Defines settings for Category's SEO
+ *
+ * @deprecated since 1.7.7, to be removed in the next major
  */
-final class SeoSettings
+final class SeoSettings extends ParentSeoSettings
```

Comment:
Doesn't that make Category SEO settings depend on Meta SEO settings? I mean shouldn't they be able to evolve independently? :thinking: